packages/commonwealth/tsconfig.build.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "rootDir": ".",
```

Comment:
If we have `rootDir` in `./tsconfig.json` why do we need it here? Same for `outDir`.

Reply:
it was creating a weird problem in vscode
not sure why, but this solved it

Reply:
Hmmm ok. Given our strong/recent inclination to simplify configuration files, I'm afraid we may forget this reasoning in the future and accidentally remove it. Is it worth investigating the VSCode issue further to try to resolve the problem at the root?